src/Packages/RetroMole.Render.Veldrid/Controller.cs

```diff
@@ -537,114 +524,108 @@ public void CreateDeviceResources()
                 new RasterizerStateDescription(FaceCullMode.None, PolygonFillMode.Solid, FrontFace.Clockwise, false, true),
                 PrimitiveTopology.TriangleList,
                 new ShaderSetDescription(vertexLayouts, new[] { _vertexShader, _fragmentShader }),
-                new ResourceLayout[] { _layout, _textureLayout },
+                new ResourceLayout[] { _mainRL, _textRL },
                 outputDescription,
                 ResourceBindingModel.Default);
             _pipeline = factory.CreateGraphicsPipeline(ref pd);
 
-            _mainResourceSet = factory.CreateResourceSet(new ResourceSetDescription(_layout,
+            _mainRS = factory.CreateResourceSet(new(
+                _mainRL,
                 _projMatrixBuffer,
-                _gd.PointSampler));
-
-            _fontTextureResourceSet = factory.CreateResourceSet(new ResourceSetDescription(_textureLayout, _fontTextureView));
+                _gd.PointSampler
+            ));
+            _ftRS = factory.CreateResourceSet(new(
+                _textRL,
+                (((Texture, TextureView))FontTexture.Texture).Item2

```

Comment:
should go back to only storing one of these, both is unneccesarry
- The original stored the tv
- The rest of the resources currently store the t